ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2309,7 +2309,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                 case StrategyTradeWorkFlowState.GoLongClosePositions:
                     if (Position.Quantity != 0)
                     {
-                        if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)

```

Comment:
This is required -  IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode  use the same paths a lot of the time-